src/coreComponents/mesh/FaceElementSubRegion.cpp

```diff
@@ -91,20 +96,21 @@ void FaceElementSubRegion::copyFromCellBlock( FaceBlockABC const & faceBlock )
   // which emphasizes the need of a refactoring.
   // In the meantime, we try to fill the face block into the sub region and hope for the best...
   {
-    auto const f = []( integer allSizes ) -> ElementType
+    auto const hack = []( integer allSizes ) -> ElementType
```

Comment:
remove?

Reply:
I actually need the `hack`. I've improved, but as along the `FaceElementSubRegion` is managed as an homogeneous subregion, I guess I'll have to do this kind of things. Maybe we can discuss the situtation?